tests/features/context/context_availability.feature

```diff
@@ -0,0 +1,36 @@
+@api
+Feature: Context availability
+  In order to check context availability
+  As a backend developer
+  I need to be able to see collection content in child entities pages.
```

Comment:
If you write this as a "backend developer" then this cannot be UAT'd. It would be much better if this was written from the perspective of a normal user of the website. This functionality actually intended for end users, so they can perform actions related to collections on content pages. Also "Context availability" is not something that will be very clear to our users.

Better would be:

"In order to be able to perform actions related to the collection to which certain content belongs
As an authenticated user
I should be presented with collection related buttons on collection content pages.